docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -205,7 +217,7 @@ In this step, you'll set up your project for use with Dagster Cloud.
 
 ---
```

Comment:
I can't comment on unchanged lines, but in line 216, could you change `dags` to `DAGs`? As this is an acronym, it should be in all caps.